lib/reader.c

```diff
@@ -421,38 +425,57 @@ CheckerInit(Checker *checker, Relation rel, TupleChecker *tchecker)
 #endif
 
         /* Set up RangeTblEntry */
-        rte = makeNode(RangeTblEntry);
```

Comment:
tab length is not correct?

Reply:
Maybe it's a tab and space issue, I changed it to space.